src/pint/models/pulsar_binary.py

```diff
@@ -377,7 +376,7 @@ def update_binary_object(self, toas, acc_delay=None):
                 # it's already in ICRS
                 updates["obs_pos"] = tbl["ssb_obs_pos"].quantity
                 updates["psr_pos"] = self._parent.ssb_to_psb_xyz_ICRS(
-                    epoch=tbl["tdbld"].astype(np.float64)
+                    epoch=tbl["tdbld"]
```

Comment:
Should this be the ld version, or the float version?

Reply:
changed.